pkg/agent/openflow/client.go

```diff
@@ -720,30 +718,31 @@ func (c *client) SendTraceflowPacket(
 	if outPort != -1 {
 		packetOutBuilder = packetOutBuilder.SetOutport(uint32(outPort))
 	}
-	packetOutBuilder = packetOutBuilder.AddLoadAction(regName, uint64(dataplaneTag), OfTraceflowMarkRange)
+	packetOutBuilder = packetOutBuilder.AddLoadAction(binding.NxmFieldIPTos, uint64(dataplaneTag), OfTraceflowMarkRange)
 
 	packetOutObj := packetOutBuilder.Done()
 	return c.bridge.SendPacketOut(packetOutObj)
 }
 
 func (c *client) InstallTraceflowFlows(dataplaneTag uint8) error {
-	flow := c.traceflowL2ForwardOutputFlow(dataplaneTag, cookie.Default)
-	if err := c.Add(flow); err != nil {
+	flows := []binding.Flow{}
+	flows = c.traceflowL2ForwardOutputFlow(dataplaneTag, cookie.Default)
```

Comment:
Could just be `flows := c.traceflowL2ForwardOutputFlow(dataplaneTag, cookie.Default)` ?

Reply:
Sure. Fixed.